vagrant/vagrantXterm.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+xterm -e 'vagrant ssh' &
```

Comment:
xterm is the most general utility I can find.  I'm happy to consider a pull request if you can find something better.